peewee_async.py

```diff
@@ -773,19 +791,16 @@ class AsyncQueryWrapper:
     def __init__(self, *, cursor=None, query=None):
         self._initialized = False
         self._cursor = cursor
-        self._result = []
+        self._rows = []
         self._result_wrapper = self._get_result_wrapper(query)
 
     def __iter__(self):
-        return iter(self._result)
-
-    def __getitem__(self, key):
-        return self._result[key]
+        while True:
+            yield self._result_wrapper.iterate()
```

Comment:
@rudyryk after this changes we can't iterate over wrapper object twice. It's big back-incompatibility! You really want to do this?

Reply:
@spumer It seems like this was fixed in #84 

Could you also take a look?

Reply:
Yea! It's fixed!